resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -27,15 +27,19 @@ private[spark] class PythonStep(
     filesDownloadPath: String) extends KubernetesSubmissionStep {
 
   override def prepareSubmission(driverSpec: KubernetesDriverSpec): KubernetesDriverSpec = {
+    val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
```

Comment:
Bash supports if-then-else. It might be tricky to get it exactly right in the Dockerfile so maybe it's not worth it.